src/shared/services/form/form-value.service.spec.ts

```diff
@@ -90,6 +90,21 @@ describe('FormValueService', () => {
       .toEqual(actual);
   })
 
+  it('should remove unnecessary fields', () => {
```

Comment:
This needs to include the name of the method that removes unnecessary fields or be wrapped in a `describe` for that method.